src/languages/ko-KR.js

```diff
@@ -518,14 +518,14 @@ export default {
     "Show update if available": "사용 가능한 경우에 업데이트 표시",
     "Also check beta release": "베타 릴리즈 확인",
     "Using a Reverse Proxy?": "리버스 프록시를 사용하시나요?",
-    "Check how to config it for WebSocket": "웹소켓에 대한 설정 방법 확인",
+    "Check how to config it for WebSocket": "웹소켓 대한 설정 방법",
     "Steam Game Server": "스팀 게임 서버",
     "Most likely causes:": "원인:",
-    "The resource is no longer available.": "더이상 사용할 수 없어요.",
+    "The resource is no longer available.": "더이상 사용할 수 없어요...",
```

Comment:
```suggestion
    "The resource is no longer available.": "더 이상 사용할 수 없어요...",
```

Check the below link.

https://www.korean.go.kr/front/onlineQna/onlineQnaView.do?mn_id=216&qna_seq=120413